engine/apps/alerts/models/alert.py

```diff
@@ -135,21 +124,36 @@ def create(
             raw_request_data=raw_request_data,
             is_the_first_alert_in_group=group_created,
         )
-
         alert.save()
+        send_alert_create_signal.apply_async((alert.pk,))
+
+        if group_created:
+            assign_labels(group, alert_receive_channel)
+            group.log_records.create(type=AlertGroupLogRecord.TYPE_REGISTERED)
+            group.log_records.create(type=AlertGroupLogRecord.TYPE_ROUTE_ASSIGNED)
+
+        if group_created or alert.group.pause_escalation:
+            # Build escalation snapshot if needed and start escalation
+            alert.group.start_escalation_if_needed(countdown=TASK_DELAY_SECONDS)
```

Comment:
Start escalation here
(instead of checking if it is the first alert https://github.com/grafana/oncall/pull/3759/files#diff-e0f70dd85cf6da1d72318bfb28759b3ebf069a28c16717ac233bf760df488a55R31)